tests/requirements-relaxed.in

```diff
@@ -0,0 +1,15 @@
+# This requirements file containes no pins on antsibull-docs and other
+# essential tools
+# It's used for testing purposes and not for the production docs.ansible.com
+# builds.
+
+jinja2 >= 3.0.0 # used by hacking/build_library/build_ansible/command_plugins/generate_man.py and dump_keywords.py
+pyyaml >= 5.1 # used by ansible-core
+resolvelib >= 0.5.3, < 1.1.0 # used by ansible-core
+sphinx
+sphinx-notfound-page
+sphinx-ansible-theme
+rstcheck
+antsibull-docs ~= 2.0
+
+-c constraints-base.in
```

Comment:
I'm not sure if it's the convention or not but I would prefer to have any referenced files before the dependencies.

Reply:
Yeah, I don't think I've ever seen this being at the end. It's usually at the beginning.

Reply:
Also, it seems like this line belongs in `constraints.in` instead of this file.

Reply:
`constraints.in` is only for `antsibull` and `sphinx` which are pinned to a specific version in `requirements.txt` but updated to the latest version by pip-compile in `requirements-relaxed.txt`. That's the whole point of this change.